packages/data/src/batch.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
I put this in `@wordpress/data` but am not sure about that.

Reply:
Yeah, seems generic enough to be used for any kind of batching exercise.

I did a bit of staring at package names, but couldn't come up with anything better. Could probably live here in data while it's maturing. 🤷

Reply:
It's all experimental and in `@wordpress/core-data` for now. We'll see how it evolves.